pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -218,6 +221,55 @@ func (r *MigPlan) ListMigrations(client k8sclient.Client) ([]*MigMigration, erro
 	return list, nil
 }
 
+// ListTemplatePods - get list of pod templates, associated with a plan resource
+func (r *MigPlan) ListTemplatePods(client k8sclient.Client) ([]corev1.Pod, error) {
+	pods := []corev1.Pod{}
+	for _, template := range DefaultTemplates {
+		// Get resources
+		list := unstructured.UnstructuredList{}
+		templateGVK := template.GroupKind().WithVersion("")
+		list.SetGroupVersionKind(templateGVK)
+		err := client.List(context.TODO(), &k8sclient.ListOptions{}, &list)
+		if err != nil {
+			return nil, err
+		}
+		resources := []unstructured.Unstructured{}
+		for _, resource := range list.Items {
+			for _, ns := range r.GetSourceNamespaces() {
+				if resource.GetNamespace() == ns {
+					resources = append(resources, resource)
+					break
+				}
+			}
+		}
+
+		for _, resource := range resources {
+			podTemplate := corev1.PodTemplateSpec{}
+			unstructuredTemplate, found, err := unstructured.NestedMap(resource.Object, template.Path()...)
+			if err != nil {
+				return nil, err
+			}
+			if !found {
+				return nil, fmt.Errorf("Template path %s was not found on resource: %s", template.TemplatePath, template.Resource)
+			}
+			err = runtime.DefaultUnstructuredConverter.FromUnstructured(unstructuredTemplate, &podTemplate)
+			if err != nil {
+				return nil, fmt.Errorf("Unable to convert resource filed '%s' to 'PodTemplateSpec': %w", template.TemplatePath, err)
```

Comment:
`%w` is new as of go 1.13 (https://blog.golang.org/go1.13-errors ). It fails in my go1.11 local environment, and our Dockerfile uses an earlier version:
```
FROM golang:1.10.3 as builder
```